src/portfolios/portfolio/components/TaskOrder/TaskOrderDetails.vue

```diff
@@ -404,6 +420,8 @@ export default class TaskOrderDetails extends Vue {
   public optionPendingClins: ClinTableRowData[] = [];
   public showInactive = false;
   public statuses = Statuses;
+  public isUpcomingTO = false;
+  public statusChipColor = '#0076A6'
```

Comment:
Instead of hardcoding this, we have a helper method `getStatusChipBgColor(status)` - Please refer to `ContributorsPanel.vue` for an example of use